src/pages/AddLiquidity/index.tsx

```diff
@@ -74,7 +75,7 @@ import approveAmountCalldata from '../../utils/approveAmountCalldata'
 import { calculateGasMargin } from '../../utils/calculateGasMargin'
 import { currencyId } from '../../utils/currencyId'
 import { maxAmountSpend } from '../../utils/maxAmountSpend'
-import { useIsRegisteredWithViolet } from '../../utils/temporary/useIsRegistered'
+import { useIsRegisteredWithViolet } from '../../utils/violet/useIsRegistered'
```

Comment:
Shouldn't the things in temp be imported through sdk?

Reply:
👍 
Let's keep "temporary" as folder name, otherwise it looks like we've accepted it to reside there.